packages/frontend/src/utils/chain-data.ts

```diff
@@ -0,0 +1,29 @@
+import { SUPPORTED_AMMS, SUPPORTED_CHAIN_ICONS } from "@/commons";
+import type { ChainData } from "@/types";
+import { ADDRESS, SupportedChain } from "@metrom-xyz/contracts";
+import { MetromApiClient, SERVICE_URLS } from "sdk";
+
+export const buildChainData = () => {
+    const environmentChains = Object.keys(
```

Comment:
Can't you take the `entry` instead of the key so that the key's value can be used in line 18 without an additional map access? Minor thing, I know.

Reply:
Oh sure, addressed in fdd28266bf94f024504f0410c4f29bf1f35a86ea.